timers: remove outdated list of available timers (#94)
runelite/wiki: Timers.md

## Timers Configuration

> The Timers plugin will show various timers in an infobox to the top left of the game window, and all timers can be switched on or off. Please refer to the config for more information.

Reviewer: ```suggestion
The Timers plugin will show various timers in an infobox, all of these timers can be switched on or off. Please refer to the Timers plugin settings for the list of available timers.
```
maybe this idk
